api/src/test/java/uk/gov/dhsc/htbhf/card/CardServicesIntegrationTest.java

```diff
@@ -32,34 +35,39 @@
 import static uk.gov.dhsc.htbhf.card.testhelper.TestConstants.DEPOSIT_REFERENCE_ID;
 
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-public class CardServicesIntegrationTest {
+@AutoConfigureWireMock(port = 8120)
+class CardServicesIntegrationTest {
 
     private static final URI ENDPOINT = URI.create("/v1/cards");
     private static final String BALANCE_URL = ENDPOINT + "/1/balance";
     private static final String DEPOSIT_URL = ENDPOINT + "/1/deposit";
-
-    private static final String CARD_SERVICES_URL = "http://localhost:8120/v1/cards";
-    private static final String CARD_SERVICES_DEPOSIT_URL = CARD_SERVICES_URL + "/1/deposit";
+    private static final String CARD_SERVICES_URL = "/v1/cards";
 
     @Autowired
     private TestRestTemplate restTemplate;
 
-    @MockBean
-    private RestTemplate restTemplateWithIdHeaders;
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @AfterEach
+    @SuppressWarnings("PMD.UnnecessaryFullyQualifiedName")
```

Comment:
Shall we turn this warning off altogether?

Reply:
Removed.